CMake/sitkLanguageOptions.cmake

```diff
@@ -15,23 +16,38 @@ sitk_check_dynamic_lookup(MODULE
   SITK_UNDEFINED_SYMBOLS_ALLOWED
   )
 
+if ( SITK_UNDEFINED_SYMBOLS_ALLOWED )
+  set( _QUIET_LIBRARY "QUIET" )
+else()
+  set( _QUIET_LIBRARY ${_QUIET} )
+endif()
+
+
 option(WRAP_DEFAULT "The default initial value for wrapping a language when it is detected on the system." ON)
 mark_as_advanced(WRAP_DEFAULT)
 
 #
-# Macro to set "_QUIET" and "_QUIET_LIBRARY" based on the first
-# argument being defined and true, to either REQUIRED or QUIET.
+# Macro to set _QUIET and ${var}_DEFAULT based on the first
+# argument and the global WRAP_DEFAULT
 #
-macro(set_QUIET var)
-  if ( DEFINED  ${var} AND ${var} )
-    set( _QUIET "REQUIRED" )
-  else()
-    set( _QUIET "QUIET" )
-  endif()
-  if ( SITK_UNDEFINED_SYMBOLS_ALLOWED )
-    set( _QUIET_LIBRARY "QUIET" )
-  else()
-    set( _QUIET_LIBRARY ${_QUIET} )
+# if ${var} is defined and true then _QUIET will be set to "REQUIRED"
+# otherwise it'll be "QUIET".
+#
+# ${var}_DEFAULT is set to "OFF" only if ${var} is already OFF, or if
+# WRAP_DEFAULT is "OFF"
+#
+macro(set_DEFAULT_QUIET var)
+  set( _QUIET "QUIET" )
+  if ( DEFINED  ${var} )
+    if ( ${var} )
+      set( _QUIET "REQUIRED" )
+    else()
+      # ${var} is OFF
+      set(${var}_DEFAULT ${var} )
```

Comment:
I suggest to set it to `OFF`  then. It facilitates reading of the code i think .